tools/run_tests/run_tests.py

```diff
@@ -440,11 +440,15 @@ def dockerfile_dir(self):
   def _get_tox_envs(self, compiler):
     """Returns name of tox environment based on selected compiler."""
     if compiler == 'default':
-      return ('py27', 'py34')
+      return ('py27', 'py34', 'py35', 'py36')
```

Comment:
@nathanielmanistaatgoogle Should we start running py35/py36 tests by default after #6962?

If so, we want to update the following to install those pythons:
tools/dockerfile/grpc_artifact_linux_x64/Dockerfile 
tools/dockerfile/grpc_artifact_linux_x86/Dockerfile
tools/dockerfile/test/python_jessie_x64/Dockerfile
templates/tools/dockerfile/python_deps.include
tools/dockerfile/test/multilang_jessie_x64/Dockerfile 

As well as updating interop/stress/performance scripts to only attempt to build python2.7 for now.